src/components/Messages/MessageList/index.js

```diff
@@ -147,14 +155,39 @@ export class MessageList extends MemoizedComponent {
 				);
 			}
 
+			const isMyMessage = (msg) => uid && msg?.u && uid === msg?.u._id;
+			const msgSequence = (msg) => {
+				const previousMessage = getPrevTextMessage(message._id);
+				const nextMessage = getNextTextMessage(message._id);
+
+				let sequence = 'mid';
+
+				if (isMyMessage(msg)) {
+					if (!previousMessage || !isMyMessage(previousMessage) || messages[i - 2]?.t) {
```

Comment:
I didn’t understand why we are using `i-2` here. Isn't it should be `i-1` for the previous message?

Reply:
for i-1 we get the **time stamp** object instead of the actual message object.
So that's why, Had to do i-2.